polaris/polaris/sep24/serializers.py

```diff
@@ -68,6 +80,9 @@ def to_representation(self, instance):
         else:  # withdrawal
             del data["deposit_memo"]
             del data["deposit_memo_type"]
+        if not self.context.get("sep6"):
```

Comment:
Why would a SEP6 object ever go through a sep24 serializer?

Reply:
The responses are identical except for these two attributes. I could create a separate serializer but I figured its not worth it since the attributes they they won't change.

Reply:
If this is shared then we should put it in a shared folder

Reply:
done